services/nutanix-ai/2.0.0/pre-install/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
pls move [nai-self-signed-cert.yaml](https://github.com/nutanix-cloud-native/nkp-nutanix-product-catalog/blob/main/services/nutanix-ai/2.0.0/nai-self-signed-cert.yaml) in pre-install

Reply:
sure. I will address this in another PR